base/exports.jl

```diff
@@ -993,6 +993,7 @@ export
     error,
     rethrow,
     retry,
+    default_retry_check,
```

Comment:
Some of those are long type names and are ok. Most of the long names with underscores we have been working to eliminate. A long underscored name is a strong indication that you are exposing insufficiently well factored concepts. This is worse for verbs than for nouns.